src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.heartbeat
+
+import akka.actor._
+import scala.concurrent.duration._
+
+// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
```

Comment:
right, well.. VI doesn't :)

On Fri, Jul 15, 2016 at 3:18 PM, Matthias Eichstedt <
notifications@github.com> wrote:

> In src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala
> <https://github.com/mesosphere/marathon/pull/4069#discussion_r71027196>:
>
> > @@ -0,0 +1,97 @@
> > +package mesosphere.marathon.core.heartbeat
> > +
> > +import akka.actor._
> > +import scala.concurrent.duration._
> > +
> > +// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
>
> I see. No big deal. We don't generate any docs so far, so that's that.
>
> In general, the way to go would be to
>
>   /**
>     * [docs here]
>     */
>
> (IntelliJ happens to generate annotations for arguments/return type and
> such if you type the /** and hit enter fwiw.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4069/files/718c11edd708bc282deea7cca641f663168b5c7c#r71027196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLMzbeIqHSzToBcSVanZgcQgC4AVbks5qV9z-gaJpZM4JGoyy>
> .
>